lib/dom/info/domDepth.js

```diff
@@ -23,7 +23,8 @@
   function numParents(elem) {
     var n = 0;
     if (elem.parentNode) {
-      while ((elem = elem.parentNode)) {
+      elem = elem.parentNode;
+      while (elem) {
```

Comment:
True, missed it :/ I will rebase tomorrow.
On Jun 3, 2016 00:48, "Tobias Lidskog" <notifications@github.com> wrote:

> In lib/dom/info/domDepth.js
> <https://github.com/sitespeedio/coach/pull/122#discussion_r65624962>:
>
> > @@ -23,7 +23,8 @@
> >    function numParents(elem) {
> >      var n = 0;
> >      if (elem.parentNode) {
> > -      while ((elem = elem.parentNode)) {
> > +      elem = elem.parentNode;
> > +      while (elem) {
>
> This will spin forever!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sitespeedio/coach/pull/122/files/ce558a5dc340044bd7535e3be263fcad9346026e#r65624962>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAVVtZTw-Tw0MrRLV8om_AmlZ3zCS65pks5qH0_EgaJpZM4Isx7O>
> .
>